src/main/java/com/lambdaschool/oktafoundation/services/HelperFunctions.java

```diff
@@ -37,4 +37,42 @@
 
 	User getCallingUser();
 
+	// THIS may be a good/simple/customizable solution to adding
```

Comment:
ALL OF THIS CODE IS MEANT TO BE COMMENTED OUT

This is an interface. So these are all simply Method Signatures for some REAL CLASS to implement. 

These are not in the scope of this PR, so I didn't want to add these changes in here... but I also didn't want to forget this as a possibility for future changes

Reply:
In fact, these additions may not even be in-scope for this month. But would still love to leave this in for some future contributor.